src/TorSharp/Tools/Tool.cs

```diff
@@ -41,5 +41,10 @@ internal class Tool
         /// The full path to the tool's configuration file.
         /// </summary>
         public string ConfigurationPath { get; set; }
+
+        /// <summary>
+        /// Automate detected in system.
```

Comment:
```suggestion
        /// The tool was automatically detected in the PATH.
```
(assuming we switch to `which`)